packages/extension-browser/src/devtools/views/add-version.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import semver = require('semver');
+
+import * as pkg from '../../../package.json';
+
+console.log('Package: ', pkg.version);
+const Version = () => {
+    const versionToShow = semver.inc(pkg.version, 'patch') || pkg.version;
```

Comment:
FYI, you can get this from `manifest.json` - see `extension-browser/src/devtools/utils/analytics.ts` which also reads the current version.

This is preferred to using `package.json` because the whole file will end up in the bundle generated by webpack. `manifest.json` already needs to be in our output so getting the version from there avoids adding extra bytes.

Reply:
Also that will avoid the need to pull in `semver` as `manifest.json` will already have been incremented appropriately as part of the build process when building for release.